cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApiImpl.java

```diff
@@ -63,7 +62,7 @@ public KafkaConnectApiImpl(Vertx vertx) {
         Buffer data = configJson.toBuffer();
         String path = "/connectors/" + connectorName + "/config";
         log.debug("Making PUT request to {} with body {}", path, configJson);
-        return withHttpClient((httpClient, result) ->
+        return HttpClientUtils.withHttpClient(vertx, new HttpClientOptions().setLogActivity(true), (httpClient, result) ->
```

Comment:
Just checking that this wasn't left over from debugging?

Reply:
AFAIK that matches what the old KafkaConnect API was setting for all HttpClientOptions? So I left it the same.